### ISSUE TITLE
 1.3.2 Headings/ Forms - Student- Accessibility Fixes - Level A - Meaningful Sequence 

### ISSUE THREAD

**Issue by michesmith** [bcgov/SIMS#3113]
**User Story**
As a student using a screen reader, I need application pages to be accessible so my tools can recognize page elements. 

**Acceptance Criteria**
- [x] Login Page - update heading tag for "Login or signup" and "Frequently Asked Questions" from (h3) to (h2)
<img width="1641" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBLzNGQkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--7859340de0d5e8e89a7e74bc1188ed511cbf747b/image.png" alt="image.png" />
<img width="1700" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBL202QkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--0b2e9982ece07818f1f0b1818c70322141ae290c/image.png" alt="image.png" />

- [x] Request a change - change "When to request change" from (h3) to (h2)
<img width="1606" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBd25HQkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--c811695a442056794aa2219f45e68a2d19a91933/image.png" alt="image.png" />

**Context:**
- These issues were uncovered through accessibility audit via CITZ. 
- Detailed Spreadsheet for Student s inserted inline and also attached here: [2024_Student Application_Audit.xlsx](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBN0xsQXc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--578821d4addd12c1babcafba5c861e1f5456bc9c/2024_Student%20Application_Audit.xlsx)
- When the sequence in which content is presented affects its meaning, a correct reading sequence caan be programmatically determined.

**The Problem**
- Heading structure not sequential, with some lines listed as headings when they are part of a list, and some headings listed as H5 directly after an H2. (Also pertains to 1.3.1 Info and Relationships). Many heading labels are missing as well.
